data/hai/hai reveal 5 philosophers.txt

```diff
@@ -0,0 +1,1082 @@
+# You as the player have one job: make sure everyone jumps in together, and don't die. During the first campaign it was really important the fights were all balanced for you to be able to tip the balance.
+# THIS mission is designed to give you a spectacle. A little visual demonstration of what a real battle could potentially look and feel like with low stakes for you. It's a visual feast, not a challenge.
+mission "Hai Reveal [B02] Join the Fleet"
+	name "Join the Fleet"
+	description "Join the fleet in <waypoint> and move on Stormhold."
+	source "Sunracer"
+	waypoint "Sumar"
+	destination "Stormhold"
+	clearance
+	to offer
+		has "Hai Reveal [B01] Information Blackout: offered"
+	on offer
+		fail "Hai Reveal [B01] Information Blackout"
+		fail "Hai Reveal [A11-B] Permanent Fleets"
+		conversation
+			`<The Syndicate security captain hands you a pad with a secure validation sequence to confirm his briefing comes from the Navy, co-signed by Xilin.>`
+			`<The Republic has agreed to accompany the Hai to precisely one world which combined intelligence indicates as the most likely place for any kidnapped Hai to be, assuming they exist. At least two pirate groups there have been positively identified being involved in kidnappings or smugglings in the past.>`
+			`<The blockades that have gone up are to secure this area of space against the secret of this exercise. The Navy, Unfettered, and Hai fleets are traveling from Mountaintop and you are to escort the Syndicate security force here directly to Sumar to meet them where you will act as the central coordination vessel.>`
+			`<There will be 2 Navy Carrier groups, a full Syndicate Special Security Detachment, the Unfettered Expeditionary Fleet, and a full Hai Taskforce. The fleet's intent is to overwhelm Stormhold and perform a thorough occupation and search for Hai. Parliament will reassess the scope of concern depending on what they find.>`
+				accept
+	npc
+		government "Syndicate"
+		personality escort coward opportunistic vindictive
+		fleet
+			names "syndicate capital"
+			variant
+				"Protector" 2
+				"Vanguard" 3
+				"Vanguard (Particle)" 3
+				"Splinter (Flamethrower Blaster)" 4
+		# This special security force has acquired outfits in the post-war condition that were surplus or black market to upgrade their Splinters.
+		# However, they haven't had the time to refit and test out new options for their Protectors or Vanguards.
+
+	npc accompany
+		government "Republic"
+		system "Ultima Thule"
+		personality escort heroic opportunistic waiting
+		fleet "Human Wormhole Guards" 20
+	npc accompany
+		government "Hai"
+		system "Ultima Thule"
+		personality escort heroic opportunistic waiting
+		fleet "Hai Wormhole Guards" 20
+	npc accompany
+		government "Hai (Friendly Unfettered)"
+		system "Ultima Thule"
+		personality escort heroic opportunistic vindictive waiting
+		fleet "Large Friendly Unfettered" 7
```

Comment:
I believe `accompany` is implicitly `save`, so any of these ships dying fails the mission.

Reply:
Well that's not what the wiki says, and is very limiting if so.

Reply:
What the wiki says: `accompany: You can only complete the mission if all members of this NPC are in the same system as you.`

Strictly, yes, this is inaccurate. The ships are required to not be disabled, which this doesn't mention, but a ship can't be in the same system as you if it's dead, even if it died in that system.

Reply:
That seems rather inconvenient if you just want to make sure all *surviving* ships of a certain set are present, but don't actually require all of them to be alive.

Reply:
I'm not sure what the use would be. Are you actually handling it differently if, for example, all the ships are lost?

Reply:
It's more about not *requiring* a separate mission to track a fleet of NPC's which are there to help and don't necessarily all need to survive.